Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "You know, I recently thought about water activities we could do together. How about scuba diving?"
```

Comment:
If you have the same, exact expression and pose multiple times in a row, you only need to specify the first time. 

I would also change up the expressions a bit since you have `3eub` 3 times in a row, and the `eub` face 5 times in a row.